bumble/sdp.py

```diff
@@ -242,11 +248,11 @@ def boolean(value: bool) -> DataElement:
         return DataElement(DataElement.BOOLEAN, value)
 
     @staticmethod
-    def sequence(value: List[DataElement]) -> DataElement:
+    def sequence(value: list[DataElement]) -> DataElement:
```

Comment:
Sequence could be better?

Reply:
Yes, definitely. I'll loosen a few other sequence/iterable types as well